console/src/pages/Auth/ClientLogin.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios'
+import { cliMateServer } from '@/consts'
+import { useAuth } from '@/api/Auth'
+import useTranslation from '@/hooks/useTranslation'
+import { Spinner } from 'baseui/spinner'
+
+export default function ClientLogin() {
+    const [t] = useTranslation()
+    const { token } = useAuth()
+    const [success, setSuccess] = useState(false)
+    const [loading, setLoading] = useState(false)
+    const [errMsg, setErrMsg] = useState('')
+    useEffect(() => {
+        if (!token) {
+            return
```

Comment:
Is it possible to login in broswer when the broswer without token?

Reply:
It relies on the user being logged in, and after the console supports logging in and jumping to the pre-login page, the experience will be smoother.

Reply:
ok, i see